WordPress/Jetpack/Classes/Utility/DataMigrator.swift

```diff
@@ -51,10 +53,25 @@ final class DataMigrator {
             completion?(.failure(.sharedUserDefaultsNil))
             return
         }
+
+        copyTodayWidgetDataToJetpack()
         BloggingRemindersScheduler.handleRemindersMigration()
         completion?(.success(()))
     }
 
+    /// Copies WP's Today Widget data (in Keychain and User Defaults) into JP.
+    ///
+    /// Both WP and JP's extensions are already reading and storing data in the same location, but in case of Today Widget,
+    /// the keys used for Keychain and User Defaults are differentiated to prevent one app overwriting the other.
+    ///
+    /// Note: This method is not private for unit testing purposes.
```

Comment:
Is it not possible to test the behaviour through the public import/export functions?

Reply:
Technically it is possible, but it would take additional effort to set up the mocks for the inner dependencies (particularly the `copyDatabase` method).

I decided to sacrifice the visibility leak for time efficiency; the most important thing is the method is covered with tests for now. I've noted this visibility leak as a tech debt here: https://github.com/wordpress-mobile/WordPress-iOS/issues/19657 and will revisit this when I have time.